src/EA.Weee.Web/Areas/AatfReturn/Mappings/ToViewModel/ReturnAndAatfToSentOnCreateSiteViewModelMap.cs

```diff
@@ -22,12 +23,20 @@ public SentOnCreateSiteViewModel Map(ReturnAndAatfToSentOnCreateSiteViewModelMap
         {
             Guard.ArgumentNotNull(() => source, source);
 
+            if (source.SiteAddressData == null)
```

Comment:
Probably need a test on this null check; the contents of the if statement are not covered